test/tsconfig-test.json

```diff
@@ -3,15 +3,7 @@
     "compilerOptions": {
         "composite": true,
         "outDir": "../",
-        "types": [
-            "mocha",
-            "chai",
-            "sinon",
-            "karma-chai",
-            "webpack-env",
-            "storybook__addon-actions",
```

Comment:
I don't know why this was included in the types...take it away and we don't have to keep around the dates `addon-action` dependency and TS seems to be OK with it...